mods/beds/functions.lua

```diff
@@ -214,8 +214,9 @@ minetest.register_on_player_receive_fields(function(player, formname, fields)
 	end
 
 	if fields.force then
-		update_formspecs(is_night_skip_enabled())
-		if is_night_skip_enabled() then
+		local is_majority = (#minetest.get_connected_players() / 2) < player_in_bed
```

Comment:
Should be in a new function to have this calculation the same wherever it is required.

Reply:
Yes. Will do if we manage to fix the problem.